docs/pages/config.mdx

```diff
@@ -17,6 +17,14 @@ The is an example of a `World` config:
 ```tsx
 import { defineWorld } from "@latticexyz/world";
 
+// Used for the module parameters, not required otherwise.
+// If you use this, make sure to run:
+//
+// pnpm install @latticexyz/common
+//
+// In the packages/contracts directory.
+import { resolveTableId } from "@latticexyz/config/register";
```

Comment:
like my previous comment, this should be imported from world (see https://mud.dev/changelog for details)